tests/test_utils/config.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import contextlib
+import functools
+
+from airflow.configuration import conf
+
+
+@contextlib.contextmanager
+def conf_vars(overrides):
```

Comment:
See https://issues.apache.org/jira/browse/AIRFLOW-4865. We already have a tests/test_utils/decorators.py that contains a mock_conf_get decorator that we should use everywhere we need to change a config value in a test. 

Could you also re-target against that Jira?

Reply:
Thanks, I updated the ticket in the commit message. It looks like the `mock_conf_get` decorator / context manager isn't used anywhere, and handles fewer cases than the decorator I added, so I'm proposing to delete it and just use the new code. I also simplified the proposed new decorator--I didn't know that `contextlib.contextmanager` produces a function that's both a decorator and a context manager.

Reply:
That was used in one or two places, I guess that got removed, but this new one is nicer!